lib/oelite/cmd/bake.py

```diff
@@ -23,8 +28,34 @@ def parse_args(options, args):
     else:
         logging.getLogger().setLevel(logging.INFO)
 
-
 def run(options, args, config):
+
+    if not options.skip_os_check:
+        distro, release = setup.get_host_distro()
+        setup_file, supported = setup.get_setup_file(config, distro, release)
+
+        if not supported:
+            print('WARNING: Host distribution (%s %s) has not been validated'%(distro, release))
+
+            if os.isatty(sys.stdin.fileno()):
+                while True:
+                    try:
+                        response = raw_input(
+                            "Do you want to continue [Y/n]? ")
+                    except KeyboardInterrupt:
+                        print
+                        response = 'n'
+                    response = response.lower()
+                    if response == '':
+                        response = 'y'
+                    if response in ('y', 'n'):
+                        print
+                        break
+
+            if response == 'n':
```

Comment:
Won't this break when stdin is not tty?  Maybe just initialize response to 'n' in an else clause to the tty if.